apps/web/app/features/page/page.tsx

```diff
@@ -17,8 +19,21 @@ export function PageInset(props: HTMLStyledProps<'div'>) {
 interface PageHeadingProps {
   title: string;
   subtitle: string;
+  disclaimer: string;
+  postSlug?: string;
 }
-export function PageHeading({ title, subtitle }: PageHeadingProps) {
+export function PageHeading({ title, subtitle, disclaimer, postSlug }: PageHeadingProps) {
```

Comment:
`Page` is incredibly generic language, if this is for the `explainers` I would call it such - definitely need something more specific than page.

Reply:
I've created a new `PostPageHeading` component for pages with headers that are represented by posts, and it accepts just a post parameter.